api/README.md

```diff
@@ -653,3 +654,62 @@ This endpoint returns the URL of a PR that is created in the wpt-metadata repo.
   ]
 }
 ```
+</details>
+
+## Browser Specific Failure
+
+### /api/bsf
+Gets the BSF data of Chrome, Firefox, Safari for the home directory.
+
+The endpoint accepts GET requests.
+
+__Parameters__
+
+__`from`__ : (Optional) RFC3339 timestamp, for which to include BSF data that occured after the given time inclusively.
+
+__`to`__ : (Optional) RFC3339 timestamp, for which to include BSF data that occured before the given time exclusively.
+
+__`experimental`__ : A boolean to return BSF data for experimental or stable runs. Defaults to false.
+
+__JSON Response__
+
```

Comment:
Add a summary "the response has three top-level fields:" and make a list. Also mention `lastUpdateRevision`.